### PR TITLE
Adding Yandex Data Streams support

### PR DESCRIPTION
**Pull request by preved911**
Hello. This PR will allow using [Yandex Data Streams](https://cloud.yandex.com/en/docs/data-streams/) for output. Can it be merged, please?

### CODE REVIEW

File: types/types.go [hunk 1]
```diff
@@ -444,15 +444,20 @@ type grafanaOutputConfig struct {
 type YandexOutputConfig struct {
 	AccessKeyID     string
 	SecretAccessKey string
+	Endpoint        string
```
Comment (Issif) [inline types/types.go:447]: Are you sure we can move this setting here? It creates a breaking change for users.

File: README.md [hunk 2]
```diff
@@ -81,6 +81,7 @@ It works as a single endpoint for as many as you want `Falco` instances :
 - [**Kafka Rest Proxy**](https://docs.confluent.io/platform/current/kafka-rest/index.html)
 - [**RabbitMQ**](https://www.rabbitmq.com/)
 - [**Azure Event Hubs**](https://azure.microsoft.com/en-in/services/event-hubs/)
+- [**Yandex Data Streams**](https://cloud.yandex.com/en/docs/data-streams/)
```
Comment (Issif) [inline README.md:84]: You also need to update the settings in the two section of the README and in config_example.yaml

File: types/types.go (hunk 1 above)
Comment (preved911) [reply types/types.go]: Yes, you are right. I misunderstood. Fixed

File: README.md (hunk 2 above)
Comment (preved911) [reply README.md:84]: Thanks. Added

File: README.md [hunk 3]
```diff
@@ -454,10 +455,14 @@ yandex:
...
-    # endpoint: "" yandex storage endpoint (default: https://storage.yandexcloud.net)
+    # endpoint: "" # yandex storage endpoint (default: https://storage.yandexcloud.net)
     # bucket: "falcosidekick" # Yandex storage, bucket name
     # prefix: "" # name of prefix, keys will have format: s3://<bucket>/<prefix>/YYYY-MM-DD/YYYY-MM-DDTHH:mm:ss.s+01:00.json
-    # minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|erro
+    # minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug
+  datastreams:
+    # endpoint: "" # Yandex Data Streams endpoint (default: https://yds.serverless.yandexcloud.net)
+    # streamName: "" # stream name in format /${region}/${folder_id}/${ydb_id}/${stream_name}
+    # minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug
```
Comment (Issif) [inline README.md:465]: Thanks. There is also the section with available ENV VARS

**Review by Issif — COMMENTED** [review]
It seems good except for the stats, a line like [this](https://github.com/falcosecurity/falcosidekick/blob/b036954ab3c1efe6f6b94f8aa0c3f24511f11864/stats.go#L66) is missing

**Comment by preved911** [comment]
> It seems good except for the stats, a line like [this](https://github.com/falcosecurity/falcosidekick/blob/b036954ab3c1efe6f6b94f8aa0c3f24511f11864/stats.go#L66) is missing

Please check it again

**Review by Issif — APPROVED** [review]
:+1:

**Pull request merged**